agent/kvs_endpoint.go

```diff
@@ -95,8 +95,8 @@ func (s *HTTPServer) KVSGetKeys(resp http.ResponseWriter, req *http.Request, arg
 	// we now are forced to check for both spellings
 	var sep string
 	params := req.URL.Query()
-	if _, ok := params["seperator"]; ok {
-		sep = params.Get("seperator")
+	if _, ok := params["separator"]; ok {
```

Comment:
This is an API break.

Reply:
Heh, we're aware of this mistake. Unfortunately, it shipped and we can't break compatibility here so if all of these related changes can be reverted that'd be very helpful.

Reply:
Dropping that is trivial (it's a single commit)